com.unity.render-pipelines.high-definition/Runtime/RenderPipeline/Camera/HDCamera.cs

```diff
@@ -237,7 +238,34 @@ internal struct VolumetricCloudsAnimationData
         internal uint                   cameraFrameCount = 0;
         internal bool                   animateMaterials;
         internal float                  lastTime;
-        internal Camera                 parentCamera = null; // Used for recursive rendering, e.g. a reflection in a scene view.
+
+        private  Camera                 m_parentCamera = null; // Used for recursive rendering, e.g. a reflection in a scene view.
+        internal  Camera                 parentCamera { get { return m_parentCamera; } }
+
+        internal void SetParentCamera(HDCamera parentHdCam)
```

Comment:
Set parent camera now is a bit more complex: it actually caches the exposure textures of the parent camera.